Added documentation for Flume integration

ChrisMandich opened the pull request:
**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27235

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

ChrisMandich committed:
Added documentation for Flume integration

klaasnicolaas commented on source/_integrations/flume.markdown:
```diff
@@ -0,0 +1,83 @@
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.99.3
```
```suggestion
ha_release: 0.101
```

klaasnicolaas committed:
:pencil2: Tweak

ChrisMandich committed:
Update source/_integrations/flume.markdown

Co-Authored-By: Klaas Schoute <klaas_schoute@hotmail.com>

klaasnicolaas reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

Happy Hacktoberfest! 🎉

ChrisMandich committed:
Update flume.markdown

Removed reference to Device ID, as this is no longer required.

ChrisMandich committed:
Update flume.markdown

Removed time zone requirements. This now uses the default Time Zone specified in HASS configuration.

frenck committed:
:pencil2: Tweaks

frenck committed:
:pencil2: Tweak

frenck committed:
:pencil2: Typo in YAML
